go-runtime/schema/common/common.go

```diff
@@ -509,6 +509,52 @@ func Deref[T types.Object](pass *analysis.Pass, node ast.Expr) (string, T) {
 	}
 }
 
+func CallExprFromVar(node *ast.GenDecl) optional.Option[*ast.CallExpr] {
+	if node.Tok != token.VAR {
+		return optional.None[*ast.CallExpr]()
+	}
+	if len(node.Specs) != 1 {
+		return optional.None[*ast.CallExpr]()
+	}
+	vs, ok := node.Specs[0].(*ast.ValueSpec)
+	if !ok {
+		return optional.None[*ast.CallExpr]()
+	}
+	if len(vs.Values) != 1 {
+		return optional.None[*ast.CallExpr]()
+	}
+	callExpr, ok := vs.Values[0].(*ast.CallExpr)
+	if !ok {
+		return optional.None[*ast.CallExpr]()
+	}
+	return optional.Some(callExpr)
+}
+
+func FuncPathEquals(pass *analysis.Pass, callExpr *ast.CallExpr, path string) bool {
```

Comment:
nit: should we add comments to this func and the one above since they're public?